docworks-dts/lib/multiple-files/builders.js

```diff
@@ -24,19 +24,12 @@ const {
 const {
 	DATASET,
 	DYNAMIC_DATASET,
-	SUB_SERVICES_KEY,
 	$W_NAME
 } = require('./constants')
+const { getServiceFilteredProperties } = require('./utils')
 
 const moduleBuilder = service => {
-	const {
-		name,
-		properties = [],
-		operations = [],
-		messages = [],
-		callbacks = [],
-		[SUB_SERVICES_KEY]: subServices = {}
-	} = service
+	const { name, messages, callbacks, operations, properties, subServices } = getServiceFilteredProperties(service)
```

Comment:
imo `getServiceFilteredProperties` is not self explanatory, maybe we should call it  `getServiceWithoutRemovedItems`?